charts/umbrella/values.yaml

```diff
@@ -691,16 +596,25 @@ dataconsumerOne:
     nameOverride: dataconsumer-1-edc
     participant:
       id: BPNL00000003AZQP
-    controlplane:
-      ssi:
-        miw:
-          url: http://managed-identity-wallets.tx.test
-          authorityId: *authority-bpn
+    iatp:
+      id: did:web:mock-util-service/BPNL00000003AZQP
+      trustedIssuers:
+        - did:web:mock-util-service/trusted-issuer
+      sts:
+        dim:
+          url: http://mock-util-service/sts
         oauth:
-          tokenurl: http://centralidp.tx.test/auth/realms/CX-Central/protocol/openid-connect/token
+          token_url: http://umbrella-centralidp:80/auth/realms/CX-Central/protocol/openid-connect/token
```

Comment:
why the change to the internal service?

Reply:
For some reason the ingress doesn't work inside the cluster of the workflow. Not sure, if this worked previously, but to get the EDC to work, I had to go back to the internal service names. 
As discussed, I changed it back to the ingress URL and use the service URL only for the testing config